StandaloneMmPkg/Library/StandaloneMmCoreEntryPoint/StandaloneMmCoreEntryPoint.inf

```diff
@@ -24,7 +24,12 @@
 [Sources.AARCH64, Sources.ARM]
   Arm/StandaloneMmCoreEntryPoint.c
   Arm/SetPermissions.c
-  Arm/CreateHobList.c
+
+[Sources.AARCH64]

```

Comment:
Same for the ARM part of this standalone MM entrypoint library. I think they are better served in ArmPkg.

Reply:
IF then.. StandaloneMmCoreEntryPoint directory there is only for x64.
But I think would make people confuse. also, whenever new architecture adds for its StandaloneMmCoreEntryPoint,
If they makes in its architecture directory, not in StandaloneMmCoreEntryPoint, the code too much spreads..
I think current way of managements (In StandaloneMmCoreEntryPoint directory saved according architecture) seems more reasonable...?

Am I missing?

Thanks

Reply:
The issue is that the code does not have much in common, if anything at all. In my understanding, Standalone MM is spec-ed to abstract the architectural differences. Instead of treating this Standalone MM package as a hodgepodge of various architectural implementations, I think it makes sense to have their own instances hosted in their corresponding packages.

It will also save you some hassle to jump hoops between ArmPkg and this one :)

Reply:
Alright, Then next update, I'll move StandaloneMmCoreEntryPoint && CpuDriver to ArmPkg.
Thanks!